pkgs/development/tools/documentation/adrdox/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, dmd }:
+
+stdenv.mkDerivation rec {
+  name = "adrdox-${version}";
+  version = "20190508";
+
+  # adrdox does not have releases or version numbers. We use dates instead,
+  # and map them to commit hashes.
+  commit = {
+    "20190508" = "e253735b1626ddb49d53f3beee9ca1d4b8ac7dda";
+  }.${version};
```

Comment:
I haven't seen this mapping from versions to commits. Is there another package that does this

Reply:
No idea, but I'll use a tag now that there is one.